README.md

```diff
@@ -1,47 +1,46 @@
-# Getting Started with Create React App and Redux
+## Description
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+This project is a mobile app that enables users to look up weather conditions around the world by city, it is built with Redux, React and the Open Weather API.
 
-## Available Scripts
+### Live link
 
-In the project directory, you can run:
+[Weather Stats](https://dapper-hummingbird-24236c.netlify.app/)
 
-### `npm start`
+## Tools
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- Javascript
+- React JS
+- Redux
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Getting Started
 
-### `npm test`
+To get a local copy up and running follow these simple steps:
 
```

Comment:
- [OPTIONAL] Please it will be nice to add a `setup section` with `steps` to clone and run your app locally, as this will be of help to none technical users who would like to know how your app works.